src/drivers/imx/sai.c

```diff
@@ -425,6 +430,7 @@ const struct dai_driver sai_driver = {
 		.probe			= sai_probe,
 		.get_handshake		= sai_get_handshake,
 		.get_fifo		= sai_get_fifo,
+		.get_srcid		= sai_get_srcid,
 		.get_hw_params		= sai_get_hw_params,
 	},
```

Comment:
This patch will no compile. Please add the patches in an order that will not break bisection.